testsuite/features/support/constants.rb

```diff
@@ -967,6 +982,11 @@
         sle-manager-tools-for-micro5-pool-x86_64-5.5
         sle-manager-tools-for-micro5-updates-x86_64-5.5
       ],
+    'sl-micro-6.0' => # TODO: Enable when it becomes available sl-manager-tools-for-micro6-updates-x86_64-6.0
+      %w[
+        sl-manager-tools-for-micro6-pool-x86_64-6.0
```

Comment:
```suggestion
        suse-manager-tools-for-sl-micro-6.0-x86_64
```
SUMA 5.0:
```bash
uyuni-server:/var/log/rhn/reposync # spacewalk-remove-channel -l
sl-micro-6.0-pool-x86_64
        suse-manager-tools-for-sl-micro-6.0-x86_64
```

Reply:
checked and corrected, will test as soon as the port is also adjusted